dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -288,20 +318,19 @@ public static void main(String[] args) {
                 DataflowPipelineWorkerPoolOptions.AutoscalingAlgorithmType.NONE);
 
         Pipeline p = Pipeline.create(pipelineOptions);
-
-        PCollectionTuple results = p.apply(TextIO.Read
-                .named("read-har")
-                .from(getHarBucket(options))
-                .withCompressionType(TextIO.CompressionType.GZIP)
-                .withCoder(HarJsonCoder.of()))
+        PCollectionTuple results = p
```

Comment:
Thank you @kennknowles!! I've updated the PR with the code provided.

The last issue is with the DataExtractorFn.processElement method. It was taking a JsonNode directly but now it takes a gzipped GcsPath file. Any advice for unzipping and converting to a JsonNode? See https://github.com/HTTPArchive/bigquery/pull/9/files#diff-81015c165ab7c57160dc04c572e5fd42R143